Libs/xlib-2.0/xlib-2.0.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
As nobody is using xlib we should either include it in the build projects or remove it.